### DISCUSSION TITLE
Problem on TrueNAS Scale when tryng to play a film it gives compatibility error from browser or FireTV

### DISCUSSION THREAD

**Discussion by AlexBarbato11** [jellyfin/jellyfin#11021] [Q&A]
Greetings to everyone, I'm a newbie with TrueNas Scale, I had this laptop that I didn't use anymore and I wanted to convert it in an home server as a personal project.

The specs:
Chuwi GemiBook Pro
Intel Celeron J4125
16GB DDR4 2133mhz
boot drive 128GB SSD sata m.2
pool drive 500GB SSD sata m.2
1Gb Ethernet interface (enx207bd2ccfa64)

Following different guides on forums and youtube I did the TrueNas scale setup, also I wanted to install some apps to host some services, one of this Jellyfin, for install I followed the truecharts guide and saw some videos for confirmation, the films and metadata are loaded, the server is being reached by devices the problem is that when I try to play any film the system gives me this error "This client isn't compatible with the media and the server isn't sending a compatible media format".

I read some posts online about this problem, but I really don't know what's causing the problem to me and what I have to check now.

**Comment by pedrojreis** [comment]
Had a friend with that problem some time ago, I believe he fixed with giving the right permissions to the media to the user - you may want to check that out :)
